Automattic/wp-calypso
# js-utils: make package private, add code owners, add important notice, change version

#### Changes proposed in this Pull Request

* Mark `@automattic/js-utils` as private and change version to `0.1.0`
* Add @Automattic/team-calypso-frameworks as code owners
* Add a note about the purpose of this package and warn about experimental status

Context: p1614937379412800-slack-C7YPUHBB2

#### Testing instructions

* Make sure tests pass
* Do the changes make sense?
* Check for spelling mistakes 😬
